src/Common/CosmosClient.ts

```diff
@@ -18,10 +18,11 @@ const _global = typeof self === "undefined" ? window : self;
 export const tokenProvider = async (requestInfo: Cosmos.RequestInfo) => {
   const { verb, resourceId, resourceType, headers } = requestInfo;
 
-  const aadDataPlaneFeatureEnabled =
-    userContext.features.enableAadDataPlane && userContext.databaseAccount.properties.disableLocalAuth;
   const dataPlaneRBACOptionEnabled = userContext.dataPlaneRbacEnabled && userContext.apiType === "SQL";
-  if (aadDataPlaneFeatureEnabled || (!userContext.features.enableAadDataPlane && dataPlaneRBACOptionEnabled)) {
+  if (
+    userContext.features.enableAadDataPlane ||
+    (!userContext.features.enableAadDataPlane && dataPlaneRBACOptionEnabled)
+  ) {
```

Comment:
Can this be simplified to (userContext.features.enableAadDataPlane || dataPlaneRBACOptionEnabled) ?

Reply:
Thanks. Addressed